packages/code-studio/src/settings/SettingsUtils.tsx

```diff
@@ -43,18 +43,18 @@ export function getFormattedVersionInfo(
   serverConfigValues: ServerConfigValues
 ): Record<string, string> {
   const ua = Bowser.parse(window.navigator.userAgent);
-  const browser = `${(ua.browser.name ?? '') || 'Unknown'} ${
-    // use only the major version and minor version, rest usually empty 120.1.0.0 -> 120.1
+  const browser = `${ua.browser.name ?? ''} ${
+    // use only the major version and minor version, rest is usually empty 120.1.0.0 -> 120.1
     Number(parseFloat(ua.browser.version ?? '')) || ''
```

Comment:
```suggestion
    Number(parseFloat(ua.browser.version ?? ''))
```

Reply:
No, the outer Number() can return NaN if the inner parseFloat returns NaN when version is undefined. And the inner ?? is needed to keep typescript happy as parseFloat is supposed to take strings, not undefined.

Reply:
Ahh yes my favorite number NaN